### DISCUSSION TITLE
Xdrip won't connect to CareLink 06/27/23

### DISCUSSION THREAD

**Discussion by KeithAmbrose** [NightscoutFoundation/xDrip#2912] [General]
On 06/27/23, CareLink was down for maintenance. Since then, I can't get xdrip to connect. Running newest build.

**Comment by Navid200** [comment]
There is already a submission to fix this:
https://github.com/NightscoutFoundation/xDrip/pull/2909

Please be patient for the review process to complete.

**Reply by KeithAmbrose** [reply]
Problem seems to be resolved with newest build! Thank you very much

**Comment by Del4x** [comment]
Same problem.
They added a page where you need to select the user who is being monitored.
This is most likely the cause of the failure.

**Reply by Navid200** [reply]
Are you using the latest Nightly release?
https://github.com/NightscoutFoundation/xDrip/releases/tag/2023.06.28

**Reply by KeithAmbrose** [reply]
I uninstalled and reinstalled newest build. Still getting "Data request error!" When I try to use source setup wizard. Thanks everyone who is working on the solutions. You all are amazing!

**Reply by Navid200** [reply]
Can you post a screenshot of the status page?  https://navid200.github.io/xDrip/docs/xDrip-Version.html

**Reply by KeithAmbrose** [reply]
Problem seems to be resolved!

**Reply by KeithAmbrose** [reply]
Thank you so much for all your hard work and support!

**Comment by Del4x** [comment]
> Are you using the latest Nightly release? https://github.com/NightscoutFoundation/xDrip/releases/tag/2023.06.28

Thanks, I didn't see this version. Everything is working :)

**Comment by 1993eko** [comment]
Hello, after updating CareLink, I got disconnected from XDrip, I downloaded the new version, but it doesn't respond, it says login error, although I write everything correctly, I received an SMS from Medtronic that the username and password are blocked due to frequent password input, can you help me?

**Reply by benceszasz** [reply]
Which country? Please paste a screenshot of the Error and Event list (you can find it in the three dots menu).

**Comment by patrikpozzolini** [comment]
I get an error message "Home toast message next: [time stamp] Task deadlock on: synctoCloudDatabases! @19"

**Comment by wtt604** [comment]
Has this issue reappeared? I am having login errors since about an hour ago

**Reply by Del4x** [reply]
i have the same again

**Reply by illuzionist** [reply]
I have a login error too. it started about 4 hours ago

**Reply by Navid200** [reply]
@benceszasz

**Reply by wtt604** [reply]
![Screenshot_20230703-160727.png](https://github.com/NightscoutFoundation/xDrip/assets/20541631/5141b162-c520-4f61-bdde-7694797c7dd9)

![Screenshot_20230703-160752.png](https://github.com/NightscoutFoundation/xDrip/assets/20541631/a5335dcd-2809-4243-9728-340776aba1ec)

I'm not sure if these images provide any help.

The carelink app is showing us the data, but nothing on xdrip.

Thanks

**Reply by wtt604** [reply]
Well, it seems like all is working now, for me anyway, I just got an upcoming low alert notification, so well done everyone involved!

**Reply by benceszasz** [reply]
There are constant CareLink system errors since the latest CareLink update. Please make sure, that:
1) There is no error displayed in the uploader application (MiniMed Mobile, Guardian, Guardian Connect)
2) Medtronic offical follower app (CareLink Connect app, Connect web app) is working without errors
3) Medtronic didn't send an official server issue/outage report email
